src/main/java/org/terasology/nui/InformativeLinksScreen.java

```diff
@@ -0,0 +1,47 @@
+package org.terasology.tutorialnui;
```

Comment:
This package declaration doesn't match the actual location of the class. It comes from the example class I think, but in here the package is simply `org.terasology.tutorialnui`

Also, our standard license header should be at the very top of the class :-)